jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/openshift/ImageStreamService.java

```diff
@@ -259,4 +259,9 @@ private Date extractDate(TagEvent tag) {
     protected static String resolveImageStreamName(ImageName name) {
         return name.getSimpleName().replace("/", "-");
     }
+
+    protected static String resolveImageStreamTagName(ImageName name) {
+        return resolveImageStreamName(name) + ":" +
```

Comment:
:question: Doesn't this change the current behavior? or was the latest tag added elsewhere before?